packages/lab/src/carousel/CarouselControls.tsx

```diff
@@ -0,0 +1,84 @@
+import {
+  Button,
+  H2,
+  SplitLayout,
+  Text,
+  makePrefixer,
+  useIcon,
+} from "@salt-ds/core";
+import { useComponentCssInjection } from "@salt-ds/styles";
+import { useWindow } from "@salt-ds/window";
+import { type HTMLAttributes, forwardRef } from "react";
+import { useCarousel } from "./CarouselContext";
+
+import carouselControlsCss from "./CarouselControls.css";
+
+const withBaseName = makePrefixer("saltCarouselControls");
+
+export interface CarouselControlsProps extends HTMLAttributes<HTMLDivElement> {}
+
+export const CarouselControls = forwardRef<
+  HTMLDivElement,
+  CarouselControlsProps
+>(function CarouselControls({ title, className, ...rest }, ref) {
+  const targetWindow = useWindow();
+  useComponentCssInjection({
+    testId: "salt-carousel-controls",
+    css: carouselControlsCss,
+    window: targetWindow,
+  });
+  const { slides, activeSlide, nextSlide, prevSlide } = useCarousel();
+  const { NextIcon, PreviousIcon } = useIcon();
+
+  const slidesCount = slides.length;
+
+  const isOnFirstSlide = activeSlide === 0;
+  const isOnLastSlide = activeSlide === slidesCount - 1;
+
+  const ControlsLabel = () => (
+    <Text as="span">
+      <strong>
+        {/* TODO: check w dev, this will need changing or a formatter*/}
```

Comment:
Probably fine for now, we should align with Pagination

Reply:
you mean secondary & no bold? pagination seem to be forcing the of in, no formatter... should there be one to allow users to format the info?